.github/workflows/bat.yml

```diff
@@ -183,4 +183,19 @@ jobs:
         run: |
           grep "::group::deploy" console.log
           grep "::error::error task failed" console.log
+          rm console.log
```

Comment:
Are the text elements in this file user-visible?

Reply:
no these are not visible